### PR TITLE
Branch1

### PR DESCRIPTION
**Pull request by warintiras**
Renamed model from Post to Position, created database table positions and deleted table posts from MySQL and related migrations.

### CODE REVIEW

File: database/migrations/2018_08_16_040629_add_newxyf_to_posts_table.php [hunk 1]
```diff
@@ -4,7 +4,7 @@
 use Illuminate\Database\Schema\Blueprint;
 use Illuminate\Database\Migrations\Migration;
 
-class AddNewxyfToPostsTable extends Migration
+class A extends Migration
```
Comment (JeremyRms) [inline database/migrations/2018_08_16_040629_add_newxyf_to_posts_table.php:7]: This is not needed

File: database/migrations/2018_08_17_050803_add_position_table.php [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+class AddPositionTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        //git
```
Comment (JeremyRms) [inline database/migrations/2018_08_17_050803_add_position_table.php:16]: This should be in another commit

**Review by JeremyRms — CHANGES_REQUESTED** [review]

File: database/migrations/2018_08_16_040629_add_newxyf_to_posts_table.php (hunk 1 above)
Comment (warintiras) [reply database/migrations/2018_08_16_040629_add_newxyf_to_posts_table.php:7]: I was having error with that so I changed the name but I deleted that migration already.

Comment (JeremyRms) [reply database/migrations/2018_08_16_040629_add_newxyf_to_posts_table.php:7]: database/migrations/2018_08_16_040629_add_newxyf_to_posts_table.php is shown as edited, not deleted.

Comment (JeremyRms) [reply database/migrations/2018_08_16_040629_add_newxyf_to_posts_table.php:7]: You'll need to use git get only the changes you need in this commit. I suggest to use a new branch to avoid to loose your work, Git can be tricky.
